pkg/bindinfo/binding.go

```diff
@@ -90,6 +96,27 @@ func (b *Binding) size() float64 {
 	return float64(res)
 }
 
+// UpdateLastUsedAt is to update binding usage info when this binding is used.
+func (b *Binding) UpdateLastUsedAt() {
+	now := time.Now()
+	b.UsageInfo.LastUsedAt.Store(&now)
+}
+
+// UpdateLastSavedAt is to update the last saved time
+func (b *Binding) UpdateLastSavedAt(ts *time.Time) {
+	b.UsageInfo.LastSavedAt.Store(ts)
+}
+
+type bindingInfoUsageInfo struct {
+	// LastUsedAt records the last time when this binding is used.
+	// It is nil if this binding has never been used or has been reset after writing into storage.
```

Comment:
> or has been reset after writing into storage.

I don't think this part is true anymore: `LastUsedAt` should only be nil if the binding has never been used.

```suggestion
	// It is nil if this binding has never been used.
```